### ISSUE TITLE
Chapter 49 References AWK Programming Language

### ISSUE THREAD

**Issue by veera-sivarajan** [remzi-arpacidusseau/ostep-typos#93]
The first reference in Chapter 49, Network File System (NFS), points to "The AWK Programming Language," which is neither related to nor mentioned anywhere in the chapter.

**Comment by remzi-arpacidusseau** [comment]
yeah, funny. Somehow the homework part used to cite it. Now I put it back! Thanks! You'll be credited in the preface shortly.
